### PR TITLE
composer.json: add branch alias

### PR DESCRIPTION
**Pull request by gharlan**
I hope this will fix https://github.com/propelorm/PropelBundle/issues/329#issuecomment-77850320.

### CODE REVIEW

**Comment by gharlan** [comment]
ping :)

**Comment by marcj** [comment]
Can you test this change using Composer's "repository" configuration?

**Comment by marcj** [comment]
This causes some issues with composer: https://travis-ci.org/propelorm/PropelBundle/jobs/53838595

File: composer.json [hunk 1]
```diff
@@ -19,5 +19,10 @@
   "autoload": {
     "psr-0": { "Propel\\PropelBundle": "" }
   },
-  "target-dir": "Propel/PropelBundle"
+  "target-dir": "Propel/PropelBundle",
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.x-dev"
```
Comment (havvg) [inline composer.json]: That's effectively not true. The master branch is obsolete.

Comment (gharlan) [inline composer.json]: Maybe we should move the 2.0 branch to master?

**Comment by gharlan** [comment]
Closing because the master branch is obsolete and it does not fix the linked issue.

**Pull request closed without merging**